alerts/openshift-virtualization-operator/KubeVirtNoAvailableNodesToRunVMs.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirtNoAvailableNodesToRunVMs
+
+## Meaning
+
+This alert fires when no nodes in the cluster have hardware virtualization or
+CPU virtualization extensions enabled.
+
+## Impact
+
+The cluster does not have the required hardware support to run virtual machines (VMs).
+
+## Diagnosis
+
+Check the hardware configuration of the nodes in the cluster.
+
+## Mitigation
+
+1. Ensure that all nodes have hardware virtualization or CPU virtualization extensions
+enabled.
+2. Check the node labels to ensure that the nodes are correctly labeled.
```

Comment:
@apinnick  Do we want to add the section about opening the case or this will be done only after we merge this?

Reply:
I could add the section about opening a case. I thought it would be unnecessary because an admin who does not know how to check node hardware config or labels probably should not be setting up a cluster. :-)

If the alert disappears after the nodes are configured correctly, we would not need a verification. If you think we need commands, I can check to see what there is.